src/api/decisionsApi.tsx

```diff
@@ -51,6 +52,7 @@ export type DecisionPost  ={
   decisionTarget: decisionTarget;
   description: string;
   date: string;
+  userid: string;
```

Comment:
Typo here
Would be better :
```suggestion
  userId: string;
```